src/gross_to_net_generation.py

```diff
@@ -436,15 +485,19 @@ def filter_gtn_conversion_factors(gtn_conversions):
         "annual_plant_ratio",
         "annual_fuel_ratio",
     ]:
+        # remove any factors that would scale net generation to less than 75% of gross generation
+        # In general, the IQR of GTN ratios is between 0.75 and 1, with an upper bound around 1.25
         # remove any ratios that are negative to avoid flipping the shape of the profile
-        factors_to_use.loc[factors_to_use[scaling_factor] < 0, scaling_factor] = np.NaN
+        factors_to_use.loc[
+            factors_to_use[scaling_factor] < 0.75, scaling_factor
+        ] = np.NaN
```

Comment:
Do these incorrect values remain in an output column somewhere so we can see where we're overwriting data-derived values with defaults? Might be useful for IDing other issues.

Reply:
Yes, these remain in the `gross_to_net_conversions_{year}.csv` file